game.py

```diff
@@ -5,35 +5,35 @@
 scissors = 'ножницы'
 paper = "бумага"
 x = "да"
-while x == "да":
+while x == "да" or x == "y":
 
-    name = input("что загадываешь? Камень ножницы или бумагу? ")
+    plrin = input("что загадываешь? Камень ножницы или бумагу? ")
 
     print('Rock, paper, scissors!')
 
     ran = random.choice([rock,paper,scissors])
     print(ran)
 
 
-    if name == rock: 
+    if plrin == rock or plrin == "r": 
         if ran == rock:
             print("Draw!")
         elif ran == scissors:
             print("I've lost!")
         elif ran == paper:
             print("You've lost!")
-    if name == scissors: 
+    if plrin == scissors or plrin == "s": 
         if ran == rock:
             print("You've lost!")
         elif ran == scissors:
             print("Draw!")
         elif ran == paper:
             print("I've lost!")
-    if name == paper:
+    if plrin == paper or plrin == "p":
         if ran == rock:
             print("I've lost!")
         elif ran == scissors:
             print("You've lost!")
         elif ran == paper:
             print("Draw!")
-    x = input("Ещё раз? да/нет ")
\ No newline at end of file
+    x = input("Ещё раз? ")
```

Comment:
May you please add, possible option s here (like `yes/no` or `y/n`)

Reply:
Sure! Check it out!